configs/autoendpoint.toml.sample

```diff
@@ -0,0 +1,107 @@
+# The URI scheme to use when referencing this server
+#scheme = "http"
+
+# The host to use
+#host = "127.0.0.1"
+
+# The port to use
+#port = 8000
+
+# The router table name
+#router_table_name = "router"
+
+# The message table name
+#message_table_name = "message"
+
+# The maximum payload size to accept in HTTP requests to this server
+#max_data_bytes = 4096
+
+# The fernet keys to use. Multiple are allowed when separated by a comma.
```

Comment:
nit: in the old python version, we had a note on how to generate this key. Many of the potential users of this package are not as savvy about how to do this. 

I wonder if it makes sense to at least include the old `autokey.py` script in the `/tools` directory. Probably should be a separate issue.

Reply:
Added: https://github.com/mozilla-services/autopush-rs/issues/217